experimenter/tests/integration/nimbus/test_cirrus_integration.py

```diff
@@ -158,3 +158,95 @@ def test_create_new_experiment_approve_remote_settings_cirrus(
     # returns the default value
     result_text_element = demo_app.wait_for_result_text(["wicked"])
     assert result_text_element.is_displayed()
+
+
+@pytest.mark.cirrus_enrollment
+def test_check_cirrus_targeting(
+    selenium,
+    experiment_url,
+    create_experiment,
+    kinto_client,
+    base_url,
+    experiment_name,
+    demo_app,
+):
+    # Launch an experiment with two branches
+    reference_branch_value = '{"enabled": true, "something": "Control branch"}'
+    treatment_branch_value = '{"enabled": true, "something": "Treatment branch"}'
+    create_experiment(
+        selenium,
+        reference_branch_value=reference_branch_value,
+        treatment_branch_value=treatment_branch_value,
+        languages=True,
+        countries=True,
+    ).launch_and_approve()
+
+    kinto_client.approve()
+
+    SummaryPage(selenium, experiment_url).open().wait_for_live_status()
+
+    home = HomePage(selenium, base_url).open()
+    assert True in [experiment_name in item.text for item in home.tables[0].experiments]
```

Comment:
This is in the other tests just to stress test the UI, but we probably don't need it here.  Waiting for the live status above should be sufficient?